writeups/vn.txt

```diff
@@ -0,0 +1,133 @@
+
+3. Has anyone threatened an appeal or otherwise indicated extreme discontent? If
+   so, please summarize the areas of conflict in separate email messages to the
+   responsible Area Director. (It should be in a separate email because this
+   questionnaire is publicly available.)
+No.
+
+4. For protocol documents, are there existing implementations of the contents of
+   the document? Have a significant number of potential implementers indicated
+   plans to implement? Are any existing implementations reported somewhere,
+   either in the document itself (as [RFC 7942][3] recommends) or elsewhere
+   (where)?
+There are multiple interoperating implementations which have been reported and tested through the typical form of social testing on the QUIC dev slack workspace. This includes major implementers who intend to use it in production.
+
+### Additional Reviews
+
+5. Do the contents of this document closely interact with technologies in other
+   IETF working groups or external organizations, and would it therefore benefit
+   from their review? Have those reviews occurred? If yes, describe which
+   reviews took place.
+No.
+
+6. Describe how the document meets any required formal expert review criteria,
+   such as the MIB Doctor, YANG Doctor, media type, and URI type reviews.
+None.
+
+7. If the document contains a YANG module, has the final version of the module
+   been checked with any of the [recommended validation tools][4] for syntax and
+   formatting validation? If there are any resulting errors or warnings, what is
+   the justification for not fixing them at this time? Does the YANG module
+   comply with the Network Management Datastore Architecture (NMDA) as specified
+   in [RFC 8342][5]?
+N/A
+
+8. Describe reviews and automated checks performed to validate sections of the
+   final version of the document written in a formal language, such as XML code,
+   BNF rules, MIB definitions, CBOR's CDDL, etc.
+Normal linting.
+
+### Document Shepherd Checks
+
+9. Based on the shepherd's review of the document, is it their opinion that this
+   document is needed, clearly written, complete, correctly designed, and ready
+   to be handed off to the responsible Area Director?
+I believe these documents are complete and clear and ready for AD review.
+
+10. Several IETF Areas have assembled [lists of common issues that their
+    reviewers encounter][6]. For which areas have such issues been identified and addressed?
+    For which does this still need to happen in subsequent reviews?
+N/A.
+
+11. What type of RFC publication is being requested on the IETF stream ([Best
+    Current Practice][12], [Proposed Standard, Internet Standard][13],
+    [Informational, Experimental or Historic][14])? Why is this the proper type
+    of RFC? Do all Datatracker state attributes correctly reflect this intent?
+Proposed Standard.
```

Comment:
The question also asks Why?

Reply:
The appropriate answer is "Because". (Snarky, because the question is a bit dumb)

Reply:
I don't disagree with the question being dumb, but if you don't answer it here it might lead to a DISCUSS so an overzealous AD can get it answered

Reply:
Note that this writeup is based on the WIP shepherd doc PR that addresses some of the community feedback levied towards the latest writeup QA template. The usefulness of this question is an unresolved thread as far as I'm concerned - see https://github.com/ietf-tools/datatracker/pull/4042#discussion_r888515151

Reply:
Fair, but saying "Because the WG reached consensus on this being the long-term VN mechanism for QUIC" doesn't hurt and might save the editors some time later :)

Reply:
sure, we aren't working to any word limit, it still boils down to "because the WG declared consensus to do so". And if that's ok, we can just make it a stock answer, which seems to defeat the purpose of the question as its currently posed.

Reply:
To be clear I'm fully in agreement with working to improve the template, and would like to see that question removed. I'm just asking for us to not be guinea pigs with this document, I don't want to have discussions about the merit of the template question during this draft's IESG review

Reply:
That's a fair concern that I take on board. We don't want to make our editors lives hard for the sake of it, nor tie up the progress of this document. 

But I'd honestly be surprised if someone objected to the statement "because we have WG consensus it is" while being happy with "because we have WG consensus that the draft $foo explains how to do $foo is appropriate"

Reply:
Oh the only difference is that it sounds slightly less snarky :-)